go.sum

```diff
@@ -67,6 +67,8 @@ github.com/bgentry/go-netrc v0.0.0-20140422174119-9fd32a8b3d3d/go.mod h1:6QX/PXZ
 github.com/bgentry/speakeasy v0.1.0/go.mod h1:+zsyZBPWlz7T6j88CTgSN5bM796AkVf0kBD4zp0CCIs=
 github.com/census-instrumentation/opencensus-proto v0.2.1/go.mod h1:f6KPmirojxKA12rnyqOA5BBL4O983OfeGPqjHWSTneU=
 github.com/cheggaaa/pb v1.0.27/go.mod h1:pQciLPpbU0oxA0h+VJYYLxO+XeDQb5pZijXscXHm81s=
+github.com/chszchen-nordcloud/go-pingdom v1.3.2-0.20210408062931-0956392a8df5 h1:StaImU1RDDHtGHPeLFuU+H123orZN7ua12jX1ipO3dY=
```

Comment:
Let's remove all references to the private lib

Reply:
Seems performing a go mod tidy can clean up unused dependencies, the private repo references have been removed. Please review.